### PR TITLE
adds ncbi-amrfinderplus v3.11.8 (same db as before: 2023-02-23.1)

### PR DESCRIPTION
**Pull request by kapsakcj**
This PR adds a dockerfile for amrfinderplus `3.11.8` & database version `2023-02-23.1`. 

The reported performance improvements from this new version look to be true, the tests implemented in the `test` stage of the dockerfile take less than 1 min to run, some even as low as 5 seconds. They used to take less than 5min or so to run in some cases.

Changes from previous dockerfile:

- ncbi-blast+ is now installed manually via binaries downloaded from NCBI FTP, instead of installing from `apt-get install` since v2.13.0 is free of a bug relating to Mac OSX that was present in v2.12.0. See the release page for detiails: https://github.com/ncbi/amr/releases/tag/amrfinder_v3.11.8
- changed PATH priority so that `amrfinder` and `blast*` commands are highest priority in PATH

I propose we name this new docker image as `staphb/ncbi-amrfinderplus:3.11.8-2023-02-23.1` (and overwrite `staphb/ncbi-amrfinderplus:latest`) in accordance with the 2 most recently-added images that include the amrfinder version followed by the database version

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [x] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [x] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ To learn more about the docker pull rate limits and the open source software pro
...
+| [NCBI AMRFinderPlus](https://hub.docker.com/r/staphb/ncbi-amrfinderplus) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/ncbi-amrfinderplus)](https://hub.docker.com/r/staphb/ncbi-amrfinderplus) | **AMRFinderPlus & database verion** <ul><li>3.1.1b</li><li>3.8.4</li><li>3.8.28</li><li>3.9.3</li><li>3.9.8</li><li>3.10.1</li><li>3.10.5</li><li>3.10.16</li><li>3.10.20</li><li>3.10.24</li><li>3.10.30</li><li>3.10.36</li><li>3.10.42</li><li>3.11.2 & 2022-12-19.1</li><li>3.11.2 & 2023-02-23.1</li><li>3.11.4 & 2023-02-23.1</li><li>3.11.8 & 2023-02-23.1</li></ul> | [https://github.com/ncbi/amr](https://github.com/ncbi/amr) |
```
Comment (erinyoung) [inline README.md:112]: Not needed for this version, but I think future versions would benefit from a drop down menu.

**Review by erinyoung — APPROVED** [review]
I don't have any changes to recommend. 

I do think that the amrfinderplus versions are going to add up, though, so in the future we should probably list those as a dropdown (like pangolin).

I am going to 
1. approve this PR
2. merge this PR
3. deploy to dockerhub and quay using the '3.11.8-2023-02-23.1' tag
4. overwrite the 'latest' tag

**Comment by erinyoung** [comment]
Update : the test isn't working right now because of connection issues to NCBI (https://github.com/StaPH-B/docker-builds/actions/runs/4683655403)

I will hopefully remember to try pushing to dockerhub and quay again tomorrow.

File: README.md (hunk 1 above)
Comment (kapsakcj) [reply README.md:112]: agreed, I can add this when the next version and/or database is released

**Comment by kapsakcj** [comment]
So the error in the GHActions workflow you linked above is a segfault - I encountered this when I first made the PR & the workflow ran for the first time. A simple re-run of the workflow resolved the issue, so I am going to re-run/re-trigger the workflow to run. Hopefully this will resolve the issue again.

I'm not sure what exactly causes this, but it might be mitigated if we reduce the `amrfinder` test commands down to 1 thread. Maybe.

**Comment by kapsakcj** [comment]
🎉 Yep, a simple re-run fixed it. New docker image has been deployed

**Pull request merged**